inventory.js

```diff
@@ -77,6 +92,12 @@ function search(input) {
 
 // Called when the DOM is ready. Loads all food items from the database.
 function loadFoodItems(queryParam) {
+  // The following three variables are used to convert the names and prices of
+  // teas and spices so that they'll be listed by ounce instead of pound.
+  const spiceRegex = /^Spices,(.)+\(BY POUND\)$/;
+  const teaRegex = /^Tea,(.)+\(BY POUND\)$/;
+  const poundToOunceDenominator = 16;
```

Comment:
Generally I would say I'd prefer constants like this to be at the top of the file. Not sure how js deals with things like that but in java I would make it a class constant